src/libs/actions/App.js

```diff
@@ -7,8 +7,7 @@ import CONST from '../../CONST';
 import Log from '../Log';
 import CONFIG from '../../CONFIG';
 import ROUTES from '../../ROUTES';
-import {printPerformanceMetrics} from '../Performance';
-import canCapturePerformanceMetrics from '../canCapturePerformanceMetrics';
+import {markEnd, markStart} from '../Performance';
```

Comment:
Should we maybe use the `import * as Performance` syntax here? 

Might be a personal preference I find it just _slightly_ clearer to understand something like this... 

```js
Timing.end(CONST.TIMING.SIDEBAR_LOADED)
Performance.markEnd(CONST.TIMING.SIDEBAR_LOADED);
```

Reply:
Yes, noticed the same, but wasn't sure about `* as Performance` 

I think maybe we should use `export default` and export a `Performance` object
It would also allow us to remove this eslint override 
```js
/* eslint-disable import/no-mutable-exports */
```

Reply:
That works for me.